core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -6046,6 +6046,12 @@ public String toString() {
               + "will drop the metadata cache of path '/mnt/alluxio-fuse/path/to/be/cleaned/'")
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey FUSE_DELETE_UNCHECKED =
+      booleanBuilder(Name.FUSE_DELETE_UNCHECKED)
+          .setDefaultValue(false)
+          .setDescription("If set, all paths will be deleted with the 'unchecked' option")
```

Comment:
can we have better user-facing description?
e.g.
when Alluxio and UFS are not in sync, Alluxio will sync metadata with UFS and then delete the file.

Reply:
Changed and made it consistent with the description of USER_FILE_DELETE_UNCHECKED.